src/transformers/file_utils.py

```diff
@@ -12,1768 +12,192 @@
-    >>> outputs = model(inputs)
-    >>> logits = outputs.logits
-    ```
-"""
-
-TF_SAMPLE_DOCSTRINGS = {
-    "SequenceClassification": TF_SEQUENCE_CLASSIFICATION_SAMPLE,
-    "QuestionAnswering": TF_QUESTION_ANSWERING_SAMPLE,
-    "TokenClassification": TF_TOKEN_CLASSIFICATION_SAMPLE,
-    "MultipleChoice": TF_MULTIPLE_CHOICE_SAMPLE,
-    "MaskedLM": TF_MASKED_LM_SAMPLE,
-    "LMHead": TF_CAUSAL_LM_SAMPLE,
-    "BaseModel": TF_BASE_MODEL_SAMPLE,
-}
-
-
-FLAX_TOKEN_CLASSIFICATION_SAMPLE = r"""
-    Example:
-
-    ```python
-    >>> from transformers import {processor_class}, {model_class}
-
-    >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
-    >>> model = {model_class}.from_pretrained("{checkpoint}")
-
-    >>> inputs = tokenizer("Hello, my dog is cute", return_tensors="jax")
-
-    >>> outputs = model(**inputs)
-    >>> logits = outputs.logits
-    ```
-"""
-
-FLAX_QUESTION_ANSWERING_SAMPLE = r"""
-    Example:
-
-    ```python
-    >>> from transformers import {processor_class}, {model_class}
-
-    >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
-    >>> model = {model_class}.from_pretrained("{checkpoint}")
-
-    >>> question, text = "Who was Jim Henson?", "Jim Henson was a nice puppet"
-    >>> inputs = tokenizer(question, text, return_tensors="jax")
-
-    >>> outputs = model(**inputs)
-    >>> start_scores = outputs.start_logits
-    >>> end_scores = outputs.end_logits
-    ```
-"""
-
-FLAX_SEQUENCE_CLASSIFICATION_SAMPLE = r"""
-    Example:
-
-    ```python
-    >>> from transformers import {processor_class}, {model_class}
-
-    >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
-    >>> model = {model_class}.from_pretrained("{checkpoint}")
-
-    >>> inputs = tokenizer("Hello, my dog is cute", return_tensors="jax")
-
-    >>> outputs = model(**inputs)
-    >>> logits = outputs.logits
-    ```
-"""
-
-FLAX_MASKED_LM_SAMPLE = r"""
-    Example:
-
-    ```python
-    >>> from transformers import {processor_class}, {model_class}
-
-    >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
-    >>> model = {model_class}.from_pretrained("{checkpoint}")
-
-    >>> inputs = tokenizer("The capital of France is {mask}.", return_tensors="jax")
-
-    >>> outputs = model(**inputs)
-    >>> logits = outputs.logits
-    ```
-"""
-
-FLAX_BASE_MODEL_SAMPLE = r"""
-    Example:
+import requests
+from filelock import FileLock
+from huggingface_hub import HfFolder, Repository, create_repo, list_repo_files, whoami
+from requests.exceptions import HTTPError
+from transformers.utils.logging import tqdm
 
-    ```python
-    >>> from transformers import {processor_class}, {model_class}
+from . import __version__
 
-    >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
-    >>> model = {model_class}.from_pretrained("{checkpoint}")
+# Backward compatibility imports, to make sure all those objects can be found in file_utils
```

Comment:
From here to line 122 is what you should read to review this PR, no code has change and this summarizes all the moves done.